.gitignore

```diff
@@ -134,4 +134,5 @@ dmypy.json
 # include small test model for tests
 !tests/assets/onnx/model.onnx
 
-.vscode
\ No newline at end of file
+.vscode
+.idea
```

Comment:
```suggestion
```
Can you remove the change on `.gitignore`? As it is irrelevant to the issue.

Reply:
Can we just leave it? Do we really need a dedicated PR for adding .idea to the .gitignore?